### ISSUE TITLE
Issues with nnlojet runcard autogeneration

### ISSUE THREAD

**Issue by scarlehoff** [NNPDF/pinefarm#73]
- [x] At the moment all the WP / WM datasets are split into a WP and WM runcard. This should only be done if there's only one observable in the set, otherwise assume that the WP and WM channels are separated.
- [ ] <s>Update the conda package to include the nnlojet python package </s>
- [x] Add multi-histogram option (this is needed for more complicated cuts)
- [ ] <s>Automatize a bit more the options-per-experiment.</s>

Please comment with other problems you find.

**Comment by achiefa** [comment]
HI @scarlehoff, I've just realised that the warmup for the VV channel didn't go through. In particular, I get a segmentation fault at the first iteration, as shown below
```
 ------------------------------------------------
 loadNextContrib_chan: loaded VV ...
 warmup...
  > > Entering legacy wrapper for NNLOJET with Vegas! < <
 Entering New Vegas
  $ OMP active
  $ Maximum number of threads:            2
  $ Number of threads selected:            2
Commencing iteration n 1
Number of events: 5000000
Segmentation fault (core dumped)
```
Note that for all the other channels I was able to complete the warmup. I've also tried to use a different dataset, but nonetheless I come across the same problem with the VV channel.

**Comment by scarlehoff** [comment]
That's strange (since all the other channels are running).
Can you send me a runcard?

(do you see the same problem if you run locally in the node?)

```
export OMP_NUM_THREADS=1
export OMP_STACKSIZE=1000M
```

Just in case this is the VV using slightly more memory per thread.

**Comment by achiefa** [comment]
I tried to run it locally in the node exporting the two env variables and I didn't get the same problem. I think this solves the problem.

**Comment by scarlehoff** [comment]
Try only with the second one. If that solves the problem then I'd recommend you use it always (you probably were close to the limit in the others they just happened to work).

If you also need the threads=1 that means there's something wrong with the installation, because I can run the VV multithreaded

**Comment by achiefa** [comment]
Yes, I'm using only the second one and it works. Many thanks @scarlehoff !
